pkgs/applications/audio/psst/default.nix

```diff
@@ -0,0 +1,38 @@
+{ pkgs, lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "psst";
+  version = "0.1.0";
+  buildAndTestSubdir = "psst-gui";
+
+  src = fetchFromGitHub {
+    owner = "jpochyla";
+    repo = pname;
+    rev = "f6eb1fe8d273af43266407d998aacac87df8c04d";
+    sha256 = "0w939w7sdalkms63dzgq6s12hwxngbhbqy7rvnq4wyn10g8hv5y1";
+  };
+
+  cargoSha256 = "1zrdd82rdshhczlyndn0g847jq0f801sj27kggyky4jxzl89p28s";
+
+  meta = with lib; {
+    description = "Fast and multi-platform Spotify client with native GUI";
+    homepage = "https://github.com/jpochyla/psst";
+    license = licenses.mit;
+    maintainers = [ maintainers.vbrandl ];
+  };
+
+  buildInputs = with pkgs; [
```

Comment:
```suggestion
  buildInputs = [
```

Reply:
Done in 32b11de94ff

Reply:
What's the reason, why you proposed this change? If I don't put `with pkgs;` before the list, I have to list all the inputs as `pkgs.pkg-config` and so on, when using `nix-build -A psst`. Is there any difference between the two or is it just best practice?